_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:50
 msgid "Be aware that while listening to the Server's signal will ensure you will be in sync with other musicians, you may also experience problems if your overall latency (indicated by the \"Delay\" light in Jamulus) is not green or at least yellow most of the time. Consult the [software manual](/wiki/Software-Manual) to understand how to adjust your setup to help with this."
-msgstr ""
+msgstr "Beachte dass das Abhören des Server Signals zwar sicherstellt, dass du mit den anderen Musikern synchron bist, dass es aber auch zu Problemen kommen kann, wenn die Gesamtlatenz (angezeigt durch die \"Delay\"-Lampe in Jamulus) die meiste Zeit über nicht grün oder zumindest gelb ist. Lies das [Software-Handbuch](/wiki/Software-Manual), um zu erfahren, wie du dein Setup in diesem Fall anpassen kannst."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:52
 msgid "### Can't work out your mic settings?"
-msgstr ""
+msgstr "### Du kannst deine Mikrofoneinstellungen nicht herausfinden?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:54
 msgid "When using a microphone while playing your other instrument, you can use a stereo audio input signal in your settings where one channel is connected to the instrument and the other channel is connected to a microphone signal. On the microphone channel an optional reverberation effect can be applied."
-msgstr ""
+msgstr "Wenn du ein Mikrofon verwendest, während du ein anderes Instrument spielst, kannst du in deinen Einstellungen ein Stereo-Audioeingangssignal verwenden, bei dem ein Kanal mit dem Instrument und der andere Kanal mit einem Mikrofonsignal verbunden ist. Auf den Mikrofonkanal kann ein optionaler Hall-Effekt eingestellt werden."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:56
 msgid "### Buffer LEDs suddenly going red, outages, jittering, weird sounds?"
-msgstr ""
+msgstr "### Puffer-LEDs leuchten plötzlich rot, Ausfälle, [Jitter](https://de.wikipedia.org/wiki/Jitter), seltsame Geräusche?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:58
 msgid "The CPU in your computer may be having issues. Try not to have anything competing with Jamulus (like Zoom Meetings or Facebook live streams) on your machine. Or at least quit them while you're playing. Prevent things like virus scanners doing scans, or software updates happening, etc. Be aware that the reverb setting in Jamulus also uses more CPU the more reverb you have."
-msgstr ""
+msgstr "Möglicherweise gibt es Probleme mit der CPU deines Computers. Versuche alles was mit Jamulus konkurriert (wie Zoom Meetings oder Facebook-Livestreams) von deinem Computer zu entfernen. Oder beende diese zumindest, während du spielst. Verhindere dass z. B. Virenscanner Scans durchführen oder Software-Updates durchgeführt werden, usw. Beachte dass auch die Hall-Einstellung in Jamulus mehr CPU verbraucht, je mehr Hall eingestellt wird."
```

Comment:
As noted in the other PR, this should be changed in the English version. Don't be so specific. Better: Your computer might have high load. Not specifically "the CPU has problems".

@gilgongo

Reply:
Done.